src/index.js

```diff
@@ -39,6 +46,55 @@ class Offline {
     this.exitCode = 0;
     this.provider = 'aws';
     this.start = this.start.bind(this);
+    this.coverageDataOutputPath = process.env.SLS_OFFLINE_COVERAGE_DATA_OUTPUT_PATH;
+    this.shouldOutputCoverage = typeof this.coverageDataOutputPath !== 'undefined';
+
+    if (this.shouldOutputCoverage) {
+      debugLog(
+        `Code coverage will be written to ${this.coverageDataOutputPath}`
+      );
+      /* eslint-disable new-cap */
```

Comment:
Can be changed to a single line disable:
```suggestion
      /* eslint-disable-next-line new-cap */
```

Reply:
I tried this initially, but for some reason eslint isn't picking up the single line disables in this repo... Might just be on my end. Could you try to reproduce this?

Reply:
Ah, rather old version of eslint, could use `eslint-disable-line` on the line in question.